src/aleph_client/asynchronous.py

```diff
@@ -690,13 +856,26 @@ async def watch_messages(
         start_date: Optional[Union[datetime, float]] = None,
         end_date: Optional[Union[datetime, float]] = None,
         session: Optional[ClientSession] = None,
-        api_server: str = settings.API_HOST,
+        api_server: Optional[str] = None,
 ) -> AsyncIterable[AlephMessage]:
     """
     Iterate over current and future matching messages asynchronously.
-    """
 
+    :param message_type: Type of message to watch
+    :param content_types: Content types to watch
+    :param refs: References to watch
+    :param addresses: Addresses to watch
+    :param tags: Tags to watch
+    :param hashes: Hashes to watch
+    :param channels: Channels to watch
+    :param chains: Chains to watch
+    :param start_date: Start date from when to watch
+    :param end_date: End date until when to watch
+    :param session: Session to use (DEFAULT: get_fallback_session())
```

Comment:
IMO you can use lowercase, `DEFAULT` everywhere is a bit aggressive on the eyes :p